packages/api/src/hooks/useMT5LoginList.ts

```diff
@@ -1,12 +1,16 @@
 import { useMemo } from 'react';
 import useActiveWalletAccount from './useActiveWalletAccount';
 import useFetch from '../useFetch';
+import useAuthorize from './useAuthorize';
 
 /** A custom hook that gets the list created MT5 accounts of the user. */
 const useMT5LoginList = () => {
+    const { isSuccess } = useAuthorize();
     const { data: wallet } = useActiveWalletAccount();
 
-    const { data: mt5_accounts, ...mt5_accounts_rest } = useFetch('mt5_login_list');
+    const { data: mt5_accounts, ...mt5_accounts_rest } = useFetch('mt5_login_list', {
+        options: { enabled: isSuccess },
+    });
```

Comment:
@hamza-deriv we don't want wait for authorize in the data layer, this should be handled in the UI part so by right when we call this hook in the UI, the authorize should be there

Reply:
I think instead of having the check isSuccess in every component using this hook, its better to have it here

Reply:
removed